lib/tool-bar-view.js

```diff
@@ -4,6 +4,11 @@ const rafDebounce = require('./raf-debounce');
 module.exports = class ToolBarView {
 
   constructor () {
+    // warning for Atom-Material-UI
+    if (atom.themes.getEnabledThemeNames()[1] === 'atom-material-ui') {
```

Comment:
I'm not fond of this check. Can we have a CSS override for `atom-material-ui` in this package if they are not willing to fix it?